docs/how-to/troubleshooting-omnitrace-linux.rst

```diff
@@ -0,0 +1,68 @@
+.. meta::
+   :description: Omnitrace documentation and reference
+   :keywords: Omnitrace, ROCm, profiler, tracking, visualization, tool, Instinct, accelerator, AMD
+
+****************************************************
+Troubleshooting Omnitrace on Linux
+****************************************************
+
+RHEL (Red Hat Enterprise Linux) distributions of Linux automatically enable a security feature 
+called SELinux that prevents `Omnitrace <https://github.com/ROCm/omnitrace>`_ from operating successfully.
+This issue applies to systems running the CentOS 9 operating system using
+AMD ROCm Software version 6.1.x alongside an MI300 GPU.
```

Comment:
6.1 and later?

an AMD Instinct MI300 GPU

Reply:
I've reached out to David regarding the affected releases (i.e. 6.1 and later, or 6.1.x).